jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/when/Authentication.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.test_generator.api.steps.when;
+
+import static com.epam.http.requests.RequestData.requestBody;
+import static com.epam.test_generator.api.BddGeneratorApi.loginUsingPOST;
+
+import com.epam.http.response.RestResponse;
+import com.epam.test_generator.api.steps.StepBackground;
+import com.epam.test_generator.controllers.user.request.LoginUserDTO;
+import cucumber.api.java.en.When;
+
+public class Authentication extends StepBackground {
+
+    @When("^I want to sign in$")
```

Comment:
"I want" is not an action. "I sign in" is better

Reply:
Fixed